pygfx/utils/text/_shaper.py

```diff
@@ -17,7 +17,7 @@
 REF_GLYPH_SIZE = 48  # 48px == 64pt
```

Comment:
It might just be a good idea to remove this variable to avoid any kind of confusion.